docs/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+<!--
+Make sure to familiarise yourself with the Runtime release process at https://github.com/polkadot-fellows/runtimes?tab=readme-ov-file#release-process 
+and the Runtime release guidelines at https://github.com/polkadot-fellows/runtimes?tab=readme-ov-file#release-guidelines.
+Use this PR template to streamline the release process, communicate on the changes that you are proposing, and provide a clear course of action for network stakeholders.
+-->
+
+## I'm submitting...
+<!---
+REQUIRED:
+Classify the type of PR your are submitting.
+-->
+  - [ ] a priority PR (i.e fixes and patches)
+  - [ ] a recommended PR (i.e enhancements, backports, migrations)
+  - [ ] an additional PR (i.e small configurations, simple migrations, general cleanup)
+
+
+### What kind of changes does this PR introduce?
+<!---
+OPTIONAL:
+Indicate if merging this PR will result in breaking changes (i.e changes to transaction/event/error encoding, polkadot-sdk migrations, or XCM and storage format), disruptions, 
+or downtimes for the network. When submitting a breaking change, please make sure to ping @SBalaguer and @anaelleltd so that they can notify ecosystem teams and builders.
+Otherwise, simply delete this section.
+-->
+
+
+### Who will be impacted by theses changes in practice?
+<!---
+OPTIONAL:
+Indicate all network stakeholders (i.e teams working on parachains, wallets, UIs, CEXes, or DEXes) that this PR will affect.
+Otherwise, simply delete this section.
+-->
+
+
+### How should impacted team/builders follow up on these changes?
+<!---
+OPTIONAL:
+Recommend a course of action (i.e resources for code refactoring or further reading) for network stakeholders.
+Otherwise, simply delete this section.
+-->
+
+## Changelog
+<!-- Remember that you can run `/merge` to enable auto-merge in the PR. -->
+
+<!-- Remember to modify the changelog and highlight all breaking changes. Otherwise, indicate if this is not needed. -->
+- [ ] The CHANGELOG entry has been submitted
```

Comment:
Why? We have CI to check it.

Reply:
> Why? We have CI to check it.

Changelog entries for breaking changes/disruptions need to stand apart from the rest. 

Currently, everything is put into generic  "Added", "Changed", "Fixed", and "Removed" sections, but there is no hierarchy.  Builders are complaining that they can't tell at a glance which changes are critical for them to know and follow up on. 

Can CI help with this?

Reply:
You cannot enforce this with a check box

Reply:
Agreed. I already updated the PR template after your initial comment.
My question now is whether CI can help with the new procedure that needs to be put in place for the Changelog, thanks.